images/nginx/rootfs/patches/nginx-1.19.3-ssl_cert_cb_yield.patch

```diff
@@ -7,7 +7,7 @@
 SSL: handled SSL_CTX_set_cert_cb() callback yielding.
 
 OpenSSL 1.0.2+ introduces SSL_CTX_set_cert_cb() to allow custom
-callbacks to serve the SSL certificiates and private keys dynamically
+callbacks to serve the SSL certificates and private keys dynamically
```

Comment:
This is actually rewriting someone's patch's commit message. It should probably be dropped. But I'm not going to change anything w/o feedback from the project.

Reply:
Please do not change files located in the patches directory. Thanks.
Also, please rebase and squash